Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -74,6 +74,25 @@ int toInt(const std::string &input, bool acceptSpaces) {
   return res;
 }
 
+unsigned int toUnsigned(const std::string &input, bool acceptSpaces) {
+  unsigned res = 0;
+  // don't need to worry about locale stuff here because
+  // we're not going to have delimiters
+
+  // sanity check on the input since strtol doesn't do it for us:
+  const char *txt = input.c_str();
+  while (*txt != '\x00') {
+    if ((*txt >= '0' && *txt <= '9') || (acceptSpaces && *txt == ' ') ||
+        *txt == '+') {
+      ++txt;
+    } else {
+      throw boost::bad_lexical_cast();
+    }
+  }
+  res = strtol(input.c_str(), nullptr, 10);
```

Comment:
I expect this should be explicitly casted to unsigned int to remove warnings.

Reply:
I'll switch to `strtoul`